client/js/_author/components/_index.jsx

```diff
@@ -1,10 +1,23 @@
 import React            from 'react';
+import { connect }            from 'react-redux';
+import * as BankActions       from '../../actions/qbank/banks';
 
-export default class Index extends React.Component {
+
+function select(state) {
+  return {
+    bank: state.banks,
+  };
+}
+export class Index extends React.Component {
```

Comment:
Its the best way to do it so additinal logic doesn't have to go into multiple components. Justin was cool with it. I'm willing to change if you have a good argument.